lib/twitter/rest/utils.rb

```diff
@@ -93,7 +93,17 @@ def perform_post_with_objects(path, options, klass)
       # @param options [Hash]
       # @param klass [Class]
       def perform_request_with_objects(request_method, path, options, klass)
-        perform_request(request_method, path, options).collect do |element|
+        result = perform_request(request_method, path, options)
```

Comment:
I think we should have additional logging inside this method as well, to get a clearer idea of what exactly the response was as soon as we got it without the twitter gem changing it.